jobfunnel/tools/tools.py

```diff
@@ -30,7 +30,6 @@ def post_date_from_relative_post_age(job_list):
             return job['date']
 
         post_date = None
-
```

Comment:
can we add this back plz :)

Reply:
Added it back on7b675c906fe448956b0a506a1e860c4d2fee1172